go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
 	gopkg.in/eapache/queue.v1 v1.1.0
 	gopkg.in/yaml.v2 v2.4.0
```

Comment:
please try to remove v2 and replace all of code to v3

Reply:
done